tests/classes/notification/PKPNotificationManagerTest.php

```diff
@@ -464,12 +478,4 @@ private function getTrivialNotification()
 
         return $notification;
     }
-
-    public function tearDown(): void
-    {
-        parent::tearDown();
-
-        // See: http://docs.mockery.io/en/latest/reference/phpunit_integration.html
```

Comment:
Are you sure this is no longer necessary? I think it's recommended when mixing PHPUnit and Mockery.

Reply:
I moved the `Mockery::close()` to the `PKPTestCase::tearDown()`.